packages/app-layout/src/vaadin-drawer-toggle.js

```diff
@@ -84,6 +93,23 @@ class DrawerToggle extends Button {
       this.dispatchEvent(new CustomEvent('drawer-toggle-click', { bubbles: true, composed: true }));
     });
   }
+
+  /** @protected */
+  ready() {
+    super.ready();
+
+    this._observer = new FlattenedNodesObserver(this, () => {
```

Comment:
Using a single observer instead of two separate `slotchange` event listeners for simplicity.